app/views/results/show.html.slim

```diff
@@ -19,6 +20,26 @@
       - %i[gross_income disposable_income capital].each
         = render("summary", section: _1, links: true) if @model.calculated?(_1)
 
+  - if end_of_journey && @model.level_of_help == "controlled" && @model.decision == "eligible" && !@check.immigration_or_asylum?
```

Comment:
I think `!@check.immigration_or_asylum?` is wrong here. The discussion on Jira is about the special case where the applicant receives asylum support, but the code is just looking at whether this is an immigration or asylum case.